src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const tBody = document.querySelector('tbody');
+
+Array.from(tBody.children).forEach((tr) => {
+  const secondTd = tr.children[1];
+
+  const lastTd = tr.lastElementChild;
+
+  const clonedTd = secondTd.cloneNode(true);
+
+  tr.insertBefore(clonedTd, lastTd);
+});
+
+function cloneColumn(wrapper, columnIndex) {
```

Comment:
Consider adding error handling to ensure that the `wrapper` element exists and contains the expected structure. This will prevent potential runtime errors if the DOM structure changes or if the selector does not match any elements.